framework/auth/core.py

```diff
@@ -1292,7 +1297,7 @@ def merge_user(self, user):
                 node.save()
 
         # - projects where the user was the creator
-        for node in user.node__created:
+        for node in user.created:
```

Comment:
Ideally the creator filed from the Node, but yeah, hold off on that. Removing the backref is the important thing. It does get used a *tiny* bit at least, so no sense causing trouble in this ticket.